test/cppsuite/test_harness/workload/thread_context.cpp

```diff
@@ -111,6 +111,19 @@ transaction_context::commit(const std::string &config)
 {
     WT_DECL_RET;
     testutil_assert(_in_txn && !_needs_rollback);
+
+    /*
```

Comment:
Do we need a FIXME to remove this section on completion of WT-9198?

Reply:
See efd54370a5f8fae9703829ea1ffb39382a5ba6c8